terraform/deployments/tfc-configuration/gcp-projects.tf

```diff
@@ -0,0 +1,30 @@
+module "gcp-ga4-analytics" {
+  source  = "alexbasista/workspacer/tfe"
+  version = "0.10.0"
+
+  organization        = var.organization
+  workspace_name      = "gcp-ga4-analytics"
+  workspace_desc      = "GCP project management for the GA4 production project"
+  workspace_tags      = ["production", "ga4-analytics-352613", "gcp"]
+  terraform_version   = var.terraform_version
+  execution_mode      = "remote"
+  working_directory   = "/terraform/deployments/ga4-analytics-352613/"
+  trigger_patterns    = ["/terraform/deployments/ga4-analytics-352613/**/*"]
+  global_remote_state = true
+
+  project_name = "govuk-infrastructure"
+  vcs_repo = {
+    identifier     = "alphagov/govuk-infrastructure"
+    branch         = "main"
+    oauth_token_id = data.tfe_oauth_client.github.oauth_token_id
+  }
+
+  team_access = {
```

Comment:
Generally we give the non-prod team read-only access so people who don't have prod access yet can still see all of our infrastructure